<launch>
<group ns="harlie">
    <node machine="mainCPU" pkg="tf" type="static_transform_publisher" name="base_laser1_broadcaster" args="0.2032 0 0.1524 0 0 0 base_link base_laser1_link 100" />
    <node machine="mainCPU" pkg="harlie_base" type="pose_broadcaster.py" name="pose_broadcaster" />
    <node machine="mainCPU" pkg="harlie_base" type="odom_translator.py" name="odom_translator" >
    </node>
    <node machine="mainCPU" pkg="sicktoolbox_wrapper" type="sicklms" name="base_laser1" respawn="true">
	<param name="port" type="string" value="/dev/ttyUSB0" />    
	<param name="baud" type="int" value="500000" />
	<param name="frame_id" type="string" value="base_laser1_link" />
        <remap from="scan" to="base_laser1_scan" />
    </node>
    <node machine="mainCPU" pkg="harlie_base" type="twist_receiver.py" name="twist_receiver" />
</group>
</launch>
